scio-redis/src/main/scala/com/spotify/scio/redis/RedisIO.scala

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2020 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.redis
+
+import com.spotify.scio.ScioContext
+import com.spotify.scio.io.{EmptyTap, EmptyTapOf, ScioIO, Tap, TapT}
+import com.spotify.scio.values.SCollection
+import org.apache.beam.sdk.io.redis.{RedisIO => BeamRedisIO}
+import org.apache.beam.sdk.values.KV
+
+import scala.jdk.CollectionConverters._
+
+sealed trait RedisIO[T] extends ScioIO[T] {
+  final override val tapT: TapT.Aux[T, Nothing] = EmptyTapOf[T]
+}
+
+case class RedisConnectionOptions(host: String, port: Int, auth: Option[String], useSsl: Boolean)
```

Comment:
I had some concerns about exposing `RedisConnectionConfiguration` to scio users, because they would need to import beam sdk package `org.apache.beam.sdk.io.redis.RedisConnectionConfiguration` and maybe in the future we will need to modify it or add something new. We can always copy/paste it to our code for sure and apply all changes, but we would need to keep the namespace/package for backward compatibility which would be weird.

Reply:
Yeah sorry I thought I had removed this comment as soon as I wrote about the custom `RedisIO` impl. With a custom one we will need our own instead of the `Beam` one so 👍 in keeping it